src/common/libsubprocess/client.c

```diff
@@ -146,6 +147,45 @@ flux_future_t *subprocess_rexec (flux_t *h,
     return NULL;
 }
 
+flux_future_t *subprocess_rexec_bg (flux_t *h,
+                                    const char *service_name,
+                                    uint32_t rank,
+                                    const flux_cmd_t *cmd,
+                                    int local_flags)
+{
+    flux_future_t *f = NULL;
+    json_t *ocmd = NULL;
+    char *topic = NULL;
+
+    if (service_name == NULL)
+        service_name = "rexec";
+
+    if (asprintf (&topic, "%s.exec", service_name) < 0) {
+        errno = ENOMEM;
```

Comment:
asprintf already sets ENOMEM on error

Reply:
Thanks, wasn't defined in the manpage so I wasn't sure that was always the case.